packages/runtime-dom/src/index.ts

```diff
@@ -100,7 +100,7 @@ export const createApp = ((...args) => {
     }
 
     // clear content before mounting
-    container.innerHTML = ''
+    container.innerHTML = unsafeToTrustedHTML('') as string
```

Comment:
This can potentially be simplified using https://developer.mozilla.org/en-US/docs/Web/API/TrustedTypePolicyFactory/emptyHTML

Reply:
After reading https://github.com/w3c/trusted-types/issues/343, I think `textContent = ''` would be an easier replacement here. Refactored.